### PR TITLE
MAB predeploy NUTNR cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
NUTNR predeploys for surface moorings, glider, AUVs
Glider, AUV NUTNRs also had previous vendor .cal files pulled during predeploy testing that we do not have csvs for yet (and associated vendor documentation not in Vault - will follow up w/ Collin to get pdfs fr vendor if possible).

### CODE REVIEW

File: calibration/NUTNRB/CGINS-NUTNRB-01092__20240314.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
....36, 273.18, 273.99, 274.8, 275.61, 276.42, 277.24, 278.05, 278.86, 279.67, 280.48, 281.3, 282.11, 282.92, 283.73, 284.54, 285.36, 286.17, 286.98, 287.79, 288.6, 289.42, 290.23, 291.04, 291.85, 292.66, 293.48, 294.29, 295.1, 295.91, 296.72, 297.53, 298.34, 299.16, 299.97, 300.78, 301.59, 302.4, 303.21, 304.02, 304.83, 305.65, 306.46, 307.27, 308.08, 308.89, 309.7, 310.51, 311.32, 312.13, 312.94, 313.75, 314.56, 315.37, 316.18, 316.99, 317.8, 318.61, 319.42, 320.23, 321.04, 321.85, 322.66, 323.47, 324.27, 325.08, 325.89, 326.7, 327.51, 328.32, 329.13, 329.93, 330.74, 331.55, 332.36, 333.16, 333.97, 334.78, 335.59, 336.39, 337.2, 338.01, 338.81, 339.62, 340.43, 341.23, 342.04, 342.84, 343.65, 344.46, 345.26, 346.07, 346.87, 347.68, 348.48, 349.29, 350.09, 350.9, 351.7, 352.5, 353.31, 354.11, 354.91, 355.72, 356.52, 357.32, 358.13, 358.93, 359.73, 360.53, 361.33, 362.14, 362.94, 363.74, 364.54, 365.34, 366.14, 366.94, 367.74, 368.54, 369.34, 370.14, 370.94, 371.74, 372.54, 373.34, 374.14, 374.94, 375.73, 376.53, 377.33, 378.13, 378.92, 379.72, 380.52, 381.31, 382.11, 382.91, 383.7, 384.5, 385.29, 386.09, 386.88, 387.68, 388.47, 389.26, 390.06, 390.85, 391.65, 392.44, 393.23, 394.02]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01092__20240314.csv:8]: spot checked - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01093__20240315.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
...3.29, 274.1, 274.92, 275.73, 276.54, 277.36, 278.17, 278.98, 279.8, 280.61, 281.42, 282.24, 283.05, 283.87, 284.68, 285.49, 286.31, 287.12, 287.93, 288.75, 289.56, 290.37, 291.19, 292, 292.81, 293.63, 294.44, 295.25, 296.07, 296.88, 297.69, 298.51, 299.32, 300.13, 300.95, 301.76, 302.57, 303.38, 304.2, 305.01, 305.82, 306.64, 307.45, 308.26, 309.07, 309.89, 310.7, 311.51, 312.32, 313.13, 313.95, 314.76, 315.57, 316.38, 317.19, 318, 318.82, 319.63, 320.44, 321.25, 322.06, 322.87, 323.68, 324.49, 325.3, 326.11, 326.92, 327.73, 328.55, 329.36, 330.17, 330.98, 331.78, 332.59, 333.4, 334.21, 335.02, 335.83, 336.64, 337.45, 338.26, 339.07, 339.88, 340.68, 341.49, 342.3, 343.11, 343.92, 344.72, 345.53, 346.34, 347.14, 347.95, 348.76, 349.57, 350.37, 351.18, 351.98, 352.79, 353.6, 354.4, 355.21, 356.01, 356.82, 357.62, 358.43, 359.23, 360.04, 360.84, 361.64, 362.45, 363.25, 364.05, 364.86, 365.66, 366.46, 367.27, 368.07, 368.87, 369.67, 370.47, 371.28, 372.08, 372.88, 373.68, 374.48, 375.28, 376.08, 376.88, 377.68, 378.48, 379.28, 380.08, 380.88, 381.68, 382.47, 383.27, 384.07, 384.87, 385.67, 386.46, 387.26, 388.06, 388.85, 389.65, 390.45, 391.24, 392.04, 392.83, 393.63, 394.42, 395.22]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01093__20240315.csv:8]: spot checked - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01102__20240315.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
....86, 274.67, 275.48, 276.3, 277.11, 277.92, 278.73, 279.55, 280.36, 281.17, 281.98, 282.8, 283.61, 284.42, 285.23, 286.05, 286.86, 287.67, 288.48, 289.3, 290.11, 290.92, 291.74, 292.55, 293.36, 294.17, 294.98, 295.8, 296.61, 297.42, 298.23, 299.05, 299.86, 300.67, 301.48, 302.3, 303.11, 303.92, 304.73, 305.54, 306.35, 307.17, 307.98, 308.79, 309.6, 310.41, 311.22, 312.04, 312.85, 313.66, 314.47, 315.28, 316.09, 316.9, 317.71, 318.53, 319.34, 320.15, 320.96, 321.77, 322.58, 323.39, 324.2, 325.01, 325.82, 326.63, 327.44, 328.25, 329.06, 329.87, 330.68, 331.49, 332.29, 333.1, 333.91, 334.72, 335.53, 336.34, 337.15, 337.96, 338.76, 339.57, 340.38, 341.19, 341.99, 342.8, 343.61, 344.42, 345.22, 346.03, 346.84, 347.64, 348.45, 349.26, 350.06, 350.87, 351.68, 352.48, 353.29, 354.09, 354.9, 355.7, 356.51, 357.31, 358.12, 358.92, 359.72, 360.53, 361.33, 362.14, 362.94, 363.74, 364.55, 365.35, 366.15, 366.95, 367.76, 368.56, 369.36, 370.16, 370.96, 371.76, 372.57, 373.37, 374.17, 374.97, 375.77, 376.57, 377.37, 378.17, 378.97, 379.77, 380.56, 381.36, 382.16, 382.96, 383.76, 384.56, 385.35, 386.15, 386.95, 387.74, 388.54, 389.34, 390.13, 390.93, 391.72, 392.52, 393.31, 394.11, 394.9, 395.7]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01102__20240315.csv:8]: looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00729__20200107.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
...72.25, 273.06, 273.88, 274.69, 275.51, 276.32, 277.13, 277.95, 278.76, 279.57, 280.39, 281.2, 282.02, 282.83, 283.64, 284.46, 285.27, 286.09, 286.9, 287.71, 288.53, 289.34, 290.16, 290.97, 291.78, 292.6, 293.41, 294.23, 295.04, 295.85, 296.67, 297.48, 298.3, 299.11, 299.92, 300.74, 301.55, 302.37, 303.18, 303.99, 304.81, 305.62, 306.44, 307.25, 308.06, 308.88, 309.69, 310.5, 311.32, 312.13, 312.95, 313.76, 314.57, 315.39, 316.2, 317.01, 317.83, 318.64, 319.45, 320.26, 321.08, 321.89, 322.7, 323.52, 324.33, 325.14, 325.95, 326.77, 327.58, 328.39, 329.2, 330.02, 330.83, 331.64, 332.45, 333.26, 334.08, 334.89, 335.7, 336.51, 337.32, 338.13, 338.94, 339.76, 340.57, 341.38, 342.19, 343, 343.81, 344.62, 345.43, 346.24, 347.05, 347.86, 348.67, 349.48, 350.29, 351.1, 351.91, 352.72, 353.52, 354.33, 355.14, 355.95, 356.76, 357.57, 358.38, 359.18, 359.99, 360.8, 361.61, 362.41, 363.22, 364.03, 364.83, 365.64, 366.45, 367.25, 368.06, 368.87, 369.67, 370.48, 371.28, 372.09, 372.89, 373.7, 374.5, 375.31, 376.11, 376.92, 377.72, 378.52, 379.33, 380.13, 380.93, 381.74, 382.54, 383.34, 384.14, 384.95, 385.75, 386.55, 387.35, 388.15, 388.95, 389.75, 390.55, 391.36, 392.16, 392.96, 393.75, 394.55]",
```
Comment (cdobs) [inline calibration/NUTNRM/CGINS-NUTNRM-00729__20200107.csv:8]: spot checked - looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00729__20240320.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
...72.25, 273.06, 273.88, 274.69, 275.51, 276.32, 277.13, 277.95, 278.76, 279.57, 280.39, 281.2, 282.02, 282.83, 283.64, 284.46, 285.27, 286.09, 286.9, 287.71, 288.53, 289.34, 290.16, 290.97, 291.78, 292.6, 293.41, 294.23, 295.04, 295.85, 296.67, 297.48, 298.3, 299.11, 299.92, 300.74, 301.55, 302.37, 303.18, 303.99, 304.81, 305.62, 306.44, 307.25, 308.06, 308.88, 309.69, 310.5, 311.32, 312.13, 312.95, 313.76, 314.57, 315.39, 316.2, 317.01, 317.83, 318.64, 319.45, 320.26, 321.08, 321.89, 322.7, 323.52, 324.33, 325.14, 325.95, 326.77, 327.58, 328.39, 329.2, 330.02, 330.83, 331.64, 332.45, 333.26, 334.08, 334.89, 335.7, 336.51, 337.32, 338.13, 338.94, 339.76, 340.57, 341.38, 342.19, 343, 343.81, 344.62, 345.43, 346.24, 347.05, 347.86, 348.67, 349.48, 350.29, 351.1, 351.91, 352.72, 353.52, 354.33, 355.14, 355.95, 356.76, 357.57, 358.38, 359.18, 359.99, 360.8, 361.61, 362.41, 363.22, 364.03, 364.83, 365.64, 366.45, 367.25, 368.06, 368.87, 369.67, 370.48, 371.28, 372.09, 372.89, 373.7, 374.5, 375.31, 376.11, 376.92, 377.72, 378.52, 379.33, 380.13, 380.93, 381.74, 382.54, 383.34, 384.14, 384.95, 385.75, 386.55, 387.35, 388.15, 388.95, 389.75, 390.55, 391.36, 392.16, 392.96, 393.75, 394.55]",
```
Comment (cdobs) [inline calibration/NUTNRM/CGINS-NUTNRM-00729__20240320.csv:8]: spot checked - looks good

File: calibration/NUTNRN/CGINS-NUTNRN-01109__20230502.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
...3, 273.43, 274.22, 275.01, 275.81, 276.6, 277.4, 278.19, 278.99, 279.78, 280.58, 281.37, 282.16, 282.96, 283.75, 284.55, 285.34, 286.14, 286.93, 287.72, 288.52, 289.31, 290.11, 290.9, 291.7, 292.49, 293.28, 294.08, 294.87, 295.67, 296.46, 297.25, 298.05, 298.84, 299.64, 300.43, 301.22, 302.02, 302.81, 303.61, 304.4, 305.19, 305.99, 306.78, 307.57, 308.37, 309.16, 309.95, 310.75, 311.54, 312.33, 313.13, 313.92, 314.71, 315.51, 316.3, 317.09, 317.88, 318.68, 319.47, 320.26, 321.05, 321.84, 322.64, 323.43, 324.22, 325.01, 325.8, 326.6, 327.39, 328.18, 328.97, 329.76, 330.55, 331.34, 332.13, 332.93, 333.72, 334.51, 335.3, 336.09, 336.88, 337.67, 338.46, 339.25, 340.04, 340.83, 341.62, 342.41, 343.19, 343.98, 344.77, 345.56, 346.35, 347.14, 347.93, 348.71, 349.5, 350.29, 351.08, 351.87, 352.65, 353.44, 354.23, 355.01, 355.8, 356.59, 357.37, 358.16, 358.95, 359.73, 360.52, 361.3, 362.09, 362.88, 363.66, 364.45, 365.23, 366.01, 366.8, 367.58, 368.37, 369.15, 369.93, 370.72, 371.5, 372.28, 373.07, 373.85, 374.63, 375.41, 376.2, 376.98, 377.76, 378.54, 379.32, 380.1, 380.88, 381.66, 382.44, 383.22, 384, 384.78, 385.56, 386.34, 387.12, 387.9, 388.68, 389.46, 390.23, 391.01, 391.79, 392.57]",
```
Comment (cdobs) [inline calibration/NUTNRN/CGINS-NUTNRN-01109__20230502.csv:8]: spot checked - looks good

File: calibration/NUTNRN/CGINS-NUTNRN-01109__20240319.csv [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
...3, 273.43, 274.22, 275.01, 275.81, 276.6, 277.4, 278.19, 278.99, 279.78, 280.58, 281.37, 282.16, 282.96, 283.75, 284.55, 285.34, 286.14, 286.93, 287.72, 288.52, 289.31, 290.11, 290.9, 291.7, 292.49, 293.28, 294.08, 294.87, 295.67, 296.46, 297.25, 298.05, 298.84, 299.64, 300.43, 301.22, 302.02, 302.81, 303.61, 304.4, 305.19, 305.99, 306.78, 307.57, 308.37, 309.16, 309.95, 310.75, 311.54, 312.33, 313.13, 313.92, 314.71, 315.51, 316.3, 317.09, 317.88, 318.68, 319.47, 320.26, 321.05, 321.84, 322.64, 323.43, 324.22, 325.01, 325.8, 326.6, 327.39, 328.18, 328.97, 329.76, 330.55, 331.34, 332.13, 332.93, 333.72, 334.51, 335.3, 336.09, 336.88, 337.67, 338.46, 339.25, 340.04, 340.83, 341.62, 342.41, 343.19, 343.98, 344.77, 345.56, 346.35, 347.14, 347.93, 348.71, 349.5, 350.29, 351.08, 351.87, 352.65, 353.44, 354.23, 355.01, 355.8, 356.59, 357.37, 358.16, 358.95, 359.73, 360.52, 361.3, 362.09, 362.88, 363.66, 364.45, 365.23, 366.01, 366.8, 367.58, 368.37, 369.15, 369.93, 370.72, 371.5, 372.28, 373.07, 373.85, 374.63, 375.41, 376.2, 376.98, 377.76, 378.54, 379.32, 380.1, 380.88, 381.66, 382.44, 383.22, 384, 384.78, 385.56, 386.34, 387.12, 387.9, 388.68, 389.46, 390.23, 391.01, 391.79, 392.57]",
```
Comment (cdobs) [inline calibration/NUTNRN/CGINS-NUTNRN-01109__20240319.csv:8]: spot checked - looks good

File: calibration/NUTNRN/CGINS-NUTNRN-01121__20230605.csv [hunk 8]
```diff
@@ -0,0 +1,8 @@
...
...273.42, 274.24, 275.06, 275.88, 276.7, 277.52, 278.34, 279.16, 279.98, 280.8, 281.62, 282.44, 283.26, 284.08, 284.9, 285.72, 286.54, 287.36, 288.18, 289, 289.82, 290.64, 291.46, 292.28, 293.09, 293.91, 294.73, 295.55, 296.37, 297.19, 298.01, 298.83, 299.64, 300.46, 301.28, 302.1, 302.92, 303.74, 304.55, 305.37, 306.19, 307.01, 307.83, 308.64, 309.46, 310.28, 311.09, 311.91, 312.73, 313.55, 314.36, 315.18, 316, 316.81, 317.63, 318.44, 319.26, 320.08, 320.89, 321.71, 322.52, 323.34, 324.15, 324.97, 325.78, 326.6, 327.41, 328.23, 329.04, 329.86, 330.67, 331.49, 332.3, 333.11, 333.93, 334.74, 335.55, 336.37, 337.18, 337.99, 338.8, 339.62, 340.43, 341.24, 342.05, 342.86, 343.68, 344.49, 345.3, 346.11, 346.92, 347.73, 348.54, 349.35, 350.16, 350.97, 351.78, 352.59, 353.4, 354.21, 355.02, 355.82, 356.63, 357.44, 358.25, 359.06, 359.86, 360.67, 361.48, 362.28, 363.09, 363.9, 364.7, 365.51, 366.31, 367.12, 367.92, 368.73, 369.53, 370.34, 371.14, 371.95, 372.75, 373.55, 374.36, 375.16, 375.96, 376.76, 377.57, 378.37, 379.17, 379.97, 380.77, 381.57, 382.37, 383.17, 383.97, 384.77, 385.57, 386.37, 387.17, 387.97, 388.77, 389.57, 390.36, 391.16, 391.96, 392.75, 393.55, 394.35, 395.14, 395.94]",
```
Comment (cdobs) [inline calibration/NUTNRN/CGINS-NUTNRN-01121__20230605.csv:8]: spot checked - looks good

File: calibration/NUTNRN/CGINS-NUTNRN-01121__20240315.csv [hunk 9]
```diff
@@ -0,0 +1,8 @@
...
...273.42, 274.24, 275.06, 275.88, 276.7, 277.52, 278.34, 279.16, 279.98, 280.8, 281.62, 282.44, 283.26, 284.08, 284.9, 285.72, 286.54, 287.36, 288.18, 289, 289.82, 290.64, 291.46, 292.28, 293.09, 293.91, 294.73, 295.55, 296.37, 297.19, 298.01, 298.83, 299.64, 300.46, 301.28, 302.1, 302.92, 303.74, 304.55, 305.37, 306.19, 307.01, 307.83, 308.64, 309.46, 310.28, 311.09, 311.91, 312.73, 313.55, 314.36, 315.18, 316, 316.81, 317.63, 318.44, 319.26, 320.08, 320.89, 321.71, 322.52, 323.34, 324.15, 324.97, 325.78, 326.6, 327.41, 328.23, 329.04, 329.86, 330.67, 331.49, 332.3, 333.11, 333.93, 334.74, 335.55, 336.37, 337.18, 337.99, 338.8, 339.62, 340.43, 341.24, 342.05, 342.86, 343.68, 344.49, 345.3, 346.11, 346.92, 347.73, 348.54, 349.35, 350.16, 350.97, 351.78, 352.59, 353.4, 354.21, 355.02, 355.82, 356.63, 357.44, 358.25, 359.06, 359.86, 360.67, 361.48, 362.28, 363.09, 363.9, 364.7, 365.51, 366.31, 367.12, 367.92, 368.73, 369.53, 370.34, 371.14, 371.95, 372.75, 373.55, 374.36, 375.16, 375.96, 376.76, 377.57, 378.37, 379.17, 379.97, 380.77, 381.57, 382.37, 383.17, 383.97, 384.77, 385.57, 386.37, 387.17, 387.97, 388.77, 389.57, 390.36, 391.16, 391.96, 392.75, 393.55, 394.35, 395.14, 395.94]",
```
Comment (cdobs) [inline calibration/NUTNRN/CGINS-NUTNRN-01121__20240315.csv:8]: spot checked - looks good

**Review by cdobs — APPROVED** [review]

File: calibration/NUTNRB/CGINS-NUTNRB-01092__20240314.csv (hunk 1 above)
Comment (reedan88) [inline calibration/NUTNRB/CGINS-NUTNRB-01092__20240314.csv:8]: Spot check - looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01093__20240315.csv (hunk 2 above)
Comment (reedan88) [inline calibration/NUTNRB/CGINS-NUTNRB-01093__20240315.csv:8]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01102__20240315.csv (hunk 3 above)
Comment (reedan88) [inline calibration/NUTNRB/CGINS-NUTNRB-01102__20240315.csv:8]: Spot check - looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00729__20200107.csv (hunk 4 above)
Comment (reedan88) [inline calibration/NUTNRM/CGINS-NUTNRM-00729__20200107.csv:8]: Spot check - looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00729__20240320.csv (hunk 5 above)
Comment (reedan88) [inline calibration/NUTNRM/CGINS-NUTNRM-00729__20240320.csv:8]: Spot check - looks good.

File: calibration/NUTNRN/CGINS-NUTNRN-01109__20230502.csv (hunk 6 above)
Comment (reedan88) [inline calibration/NUTNRN/CGINS-NUTNRN-01109__20230502.csv:8]: Spot check - looks good.

File: calibration/NUTNRN/CGINS-NUTNRN-01109__20240319.csv (hunk 7 above)
Comment (reedan88) [inline calibration/NUTNRN/CGINS-NUTNRN-01109__20240319.csv:8]: Spot check - looks good.

File: calibration/NUTNRN/CGINS-NUTNRN-01121__20230605.csv (hunk 8 above)
Comment (reedan88) [inline calibration/NUTNRN/CGINS-NUTNRN-01121__20230605.csv:8]: Spot check - looks good.

File: calibration/NUTNRN/CGINS-NUTNRN-01121__20240315.csv (hunk 9 above)
Comment (reedan88) [inline calibration/NUTNRN/CGINS-NUTNRN-01121__20240315.csv:8]: Spot check - looks good.

**Review by reedan88 — APPROVED** [review]
Looks good.

**Pull request merged**